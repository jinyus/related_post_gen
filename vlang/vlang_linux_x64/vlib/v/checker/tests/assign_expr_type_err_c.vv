fn main() {
	mut foo := 'hello'
	foo *= 10
	_ = foo
}
