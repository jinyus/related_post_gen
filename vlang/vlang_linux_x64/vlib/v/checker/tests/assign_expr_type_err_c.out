vlib/v/checker/tests/assign_expr_type_err_c.vv:3:2: error: operator *= not defined on left operand type `string`
    1 | fn main() {
    2 |     mut foo := 'hello'
    3 |     foo *= 10
      |     ~~~
    4 |     _ = foo
    5 | }
